static/app/components/workflowEngine/form/automationBuilderRowLine.tsx

```diff
@@ -1,9 +1,14 @@
 import {Flex} from 'sentry/components/container/flex';
 import {space} from 'sentry/styles/space';
 
-export function RowLine({children}: {children: React.ReactNode}) {
+interface Props {
+  children: React.ReactNode;
+  className?: string;
+}
+
+export function RowLine({children, className}: Props) {
   return (
-    <Flex align="center" gap={space(1)} wrap="wrap">
+    <Flex align="center" gap={space(1)} wrap="wrap" className={className}>
       {children}
     </Flex>
```

Comment:
these three styles aren't really worth splitting into a shared component
```
display: flex;
gap: space(1)
flex-wrap: wrap;
```

Reply:
fixed it to be a styled div since a lot of components / data conditions use this styling